<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drtaccees</class>
 <widget class="QDialog" name="drtaccees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_a">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>21</y>
     <width>651</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_dr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_dr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenon_de">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_n_dr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lieu_t">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="salaire">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="grade_l">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mot_dr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gender_r">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>date_n</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>lieu_t</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>telephone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>salaire</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>grade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>mot de passe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>gender</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_dr">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>620</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifier">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>620</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
